src/kontrol/kdist/foundry.md

```diff
@@ -1,85 +1,12 @@
-  The best options are:
-   - `--reinit` - want to start over from scratch,
-   - `--no-simplify-init` - do not want to invoke simplification on all the original nodes, can be faster,
-   - `--max-depth` - increase the space between saved nodes; bigger is faster,
-   - `--max-iterations` - maximum number of nodes to store in KCFG before giving on attempting proof for that run,
-   - `--break-every-step` - save a state every opcode, slow, good for debugging,
-   - `--break-on-calls` - save a state every time a call is made, good to turn on.
-   - `--verbose` - output what the prover is doing to make sure it's making progress.
-
-- `kevm foundry-show`: Display the given KCFG so far as text.
-  Options are:
-   - `--no-minimize` - do not omit all the gory details,
-   - `--node` - can be a repeated option, display more information about a given node hash,
-   - `--node-delta` - displays the delta between two given nodes in the KCFG.
-
-- `kevm foundry-view-kcfg`: Launch the more interactive exploration of the KCFG (can be done while exploration is running, must Ctrl-C + relaunch to view updates to KCFG).
-   - The interactive KCFG puts your terminal in *application mode*. To select text in this mode, hold the modifier key provided by your terminal emulator (typically SHIFT or OPTION) while clicking and dragging. Refer to the [Textualize documentation](https://github.com/Textualize/textual/blob/main/FAQ.md#how-can-i-select-and-copy-text-in-a-textual-app) for more information.
-
-- `kevm foundry-section-edge`: Given an edge in the graph, cut it into sections to get intermediate nodes.
-
-- `kevm foundry-step-node`: Step from a given node, adding it to the CFG.
-
-- `kevm foundry-simplify-node`: Simplify a given node, and potentially replace it.
-
-Example Usage
--------------
-
-The first step is to ensure the Solidity codebase is compiled and the `out/` directory is generated.
-
-For example, in the root of this repository, you can run:
-
-*Build Foundry Project:*
-
-```sh
-$ cd tests/foundry
-$ forge build
-```
-
-*Kompile to generate K specifications:*
-
-```sh
-$ kevm foundry-kompile
-```
-
-*And discharge some specific test as a proof obligation (inside virtual environment):*
-
-```sh
-$ kevm foundry-prove --test AssertTest.test_assert_true
-```
-
-Foundry Module for KEVM
------------------------
+Foundry Module for Kontrol
+---------------------------
 
 Foundry's testing framework provides a series of cheatcodes so that developers can specify what situation they want to test.
-This file describes the KEVM specification of the Foundry testing framework, which includes the definition of said cheatcodes and what does it mean for a test to pass.
+This file describes the Kontrol specification of the Foundry testing framework, which includes the definition of said cheatcodes and what does it mean for a test to pass.
```

Comment:
This might be better.

```suggestion
This file describes the K semantics of the Foundry testing framework, which includes the definition of said cheatcodes and what does it mean for a test to pass.
```